appveyor.yml

```diff
@@ -6,7 +6,7 @@ clone_folder: c:\gopath\src\go.opencensus.io
 
 environment:
   GOPATH: c:\gopath
-  GOVERSION: 1.8
+  GOVERSION: 1.6
```

Comment:
Any way to add 1.9 as well?

Reply:
Unfortunately as I had commented in the previous PR #228, I spent a lot of time trying to do so but didn't get anywhere so I figured someone else with more knowledge of appveyor could help out.